homeassistant/components/connectedcars/const.py

```diff
@@ -0,0 +1,116 @@
+"""Constants for the ConnectedCars.io integration."""
+
+ATTRIBUTION = "Data provided by ConnectedCars.io"
+
+ATTR_API_USER_EMAIL = "email"
+ATTR_API_USER_FIRSTNAME = "first_name"
+ATTR_API_USER_LASTNAME = "last_name"
+
+ATTR_API_VEHICLE_FUELLEVEL = "vehicle_fuel_level"
+ATTR_API_VEHICLE_FUELPERCENTAGE = "vehicle_fuel_percentage"
+ATTR_API_VEHICLE_ID = "vehicle_id"
+ATTR_API_VEHICLE_LICENSEPLATE = "vehicle_license_plate"
+ATTR_API_VEHICLE_MAKE = "vehicle_make"
+ATTR_API_VEHICLE_MODEL = "vehicle_model"
+ATTR_API_VEHICLE_NAME = "vehicle_name"
+ATTR_API_VEHICLE_ODOMETER = "vehicle_odometer"
+ATTR_API_VEHICLE_POS_LATITUDE = "vehicle_position_latitude"
+ATTR_API_VEHICLE_POS_LONGITUDE = "vehicle_position_longitude"
+ATTR_API_VEHICLE_VIN = "vehicle_vin"
+ATTR_API_VEHICLE_VOLTAGE = "vehicle_voltage"
+
+ATTR_ICON = "icon"
+ATTR_IDENTIFIERS = "identifiers"
+ATTR_LABEL = "label"
+ATTR_MANUFACTURER = "manufacturer"
+ATTR_MODEL = "model"
+ATTR_UNIT = "unit"
+
+COMPLETE_QUERY = """query User {
```

Comment:
The query strings are protocol level information and should be part of a 3rd party library published on pypi instead.

Reply:
Thank you @MartinHjelmare.
I am not the maintainer of the connectedcars pypi package, I was just fortunate enough that someone else have made it and I actually enjoyed the freedom of writing the GraphQL query directly in the HA integration, because it will be simpler to maintain if we want to add more features in the future.
But if this is a choice set in stone in the HA design guidelines, then I may need to write my own version of the connectedcars pypi package. Again a greater task then envisioned.

Reply:
Please make a PR to the library repo and add them there. There's already one query string constant there.

https://github.com/niklascp/connectedcars-python/blob/89a3bd2069c7eee595c3b4821ec5d23470f9347e/connectedcars/constants.py

Reply:
Still waiting for a response on PR to add constants to original pypi package: https://github.com/niklascp/connectedcars-python/pull/2